tests/comparisons/Plugin/SimpleExample/tests/bootstrap.php

```diff
@@ -38,3 +38,18 @@
 
     return;
 }
+
+/**
+ * Load schema from a SQL dump file.
+ *
+ * If your plugin does not use database fixtures you can
+ * safely delete this.
+ *
+ * If you want to support multiple databases, consider
+ * using migrations to provide schema for your plugin,
+ * and using \Migrations\TestSuite\Migrator to load schema.
+ */
+use Cake\TestSuite\Schema\SchemaManager;
+
+// Load a schema dump file.
+SchemaManager::create('test', './tests/schema.sql');
```

Comment:
```suggestion
SchemaManager::create('test', 'tests/schema.sql');
```
Is the `./` required for any sort of security? I think we should just use relative paths from base like most places.